myriad-scheduler/src/main/java/com/ebay/myriad/configuration/ServiceConfiguration.java

```diff
@@ -129,4 +142,96 @@ public String getServiceOpts() {
     return serviceOptsName;
   }
 
+  /**
+   * Custom Serializer
+   * @param <T>
+   */
+  public static class OptionalSerializer<T> extends JsonSerializer<Optional<T>> {
+
+    private static final JsonFactory jsonFactory = new ObjectMapper().getJsonFactory();
+
+    protected ObjectMapper objMapper;
+    
+    public OptionalSerializer() {
+      objMapper = new ObjectMapper(jsonFactory);
+    }
+    
+    @Override
+    public void serialize(Optional<T> value,
+        org.codehaus.jackson.JsonGenerator jgen, SerializerProvider provider)
+        throws IOException, org.codehaus.jackson.JsonProcessingException {
+      if (value.isPresent()) {
+        objMapper.writeValue(jgen, value.get());
+      } else {
+        objMapper.writeValue(jgen, "value is absent");
+      }
+    }
+  }
+  
+  /**
+   * Custom String serializer
+   *
+   */
+  public static class OptionalSerializerStr extends OptionalSerializer<String> {
```

Comment:
Will be good to rename the class to "OptionalSerializerString", to be consistent with the names of the other classes that you've defined.